docker/Dockerfile

```diff
@@ -33,9 +33,8 @@ EXPOSE $PORT
 # List wheel files for debugging purposes
 RUN ls -la /app/dist/
 
-# Install the core Arcade packages (each has its own version)
+# Install the CLI and worker package
 RUN python -m pip install \
-    /app/dist/arcade_core-*.whl \
```

Comment:
since arcade-core is a transitive dependency through arcade-serve, I don't think we need to explicitly install it here.